app/models/rules/sealed_bid.rb

```diff
@@ -0,0 +1,53 @@
+module Rules
+  class SealedBid < Rules::Basic
+    def winning_bid
+      return Presenter::Bid::Null.new if auction.available?
```

Comment:
Well, this is wrapped in Presenter::Auction so it's not in the AR model level if that helps.